src/main/ApplicationImpl.cpp

```diff
@@ -1026,11 +1026,12 @@ ApplicationImpl::postOnMainThread(std::function<void()>&& f, std::string&& name,
 {
     LogSlowExecution isSlow{name, LogSlowExecution::Mode::MANUAL,
                             "executed after"};
-    mVirtualClock.postAction([ this, f = std::move(f), isSlow ]() {
-        mPostOnMainThreadDelay.Update(isSlow.checkElapsedTime());
-        f();
-    },
-                             std::move(name), type);
+    mVirtualClock.postAction(
+        [ this, f = std::move(f), isSlow = std::move(isSlow) ]() {
```

Comment:
c++ trivia.

I think you're actually making a copy here:
as `LogSlowExecution` has a destructor, it does not have a "good" default move constructor, so instead this will just create a new one.

Reply:
Great. More trivia for the "why does C++ even try" bucket. I'll remove the change.